loaders/src/glTF/glTFFileLoader.ts

```diff
@@ -634,13 +637,24 @@ export class GLTFFileLoader implements IDisposable, ISceneLoaderPluginAsync, ISc
 
     /** @hidden */
     public canDirectLoad(data: string): boolean {
-        return data.indexOf("asset") !== -1 && data.indexOf("version") !== -1;
+        return (data.indexOf("asset") !== -1 && data.indexOf("version") !== -1) ||
+                StringTools.StartsWith(data, "data:base64," + GLTFFileLoader.magicBase64Encoded);

```

Comment:
There may also be a mime type in for a [data URI](https://en.wikipedia.org/wiki/Data_URI_scheme#Syntax)? Do we need to handle them?

Reply:
After decoding, the base64 string should be binary data: so, should we support something other than "application/octet-stream"?

If not, I can add the check for it, else I would need some more info on what should be supported (and possibly how).

Reply:
GLB has its own mime type, see https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#mime-type. I think we should support this and `application/octet-stream`?

One other random thought. Should we allow a glTF JSON encoded as base64 here? I say no, but just wanted to call it out.

Reply:
Check of mime types added.

I don't think we should allow glTF JSON encoded base64 here either, @sebavan to decide.

Reply:
As you both vote for no. I ll definitely go with you :-) I have no real preferences either.